server/core/src/main/java/org/infinispan/server/core/LifecycleCallbacks.java

```diff
@@ -25,9 +31,15 @@
  */
 @InfinispanModule(name = "server-core", requiredModules = "core")
 public class LifecycleCallbacks implements ModuleLifecycle {
+   public static final String SERVER_STATE_CACHE = "___server_state";
```

Comment:
I think we decided to use the format `org.infinispan.CACHE_NAME` to name our internal caches...